.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
+        {
+            "type": "node",
+            "name": "vscode-jest-tests",
+            "request": "launch",
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "disableOptimisticBPs": true,
+            "cwd": "${workspaceFolder}",
+            "runtimeExecutable": "yarn",
+            "args": ["jest", "--runInBand", "--watchAll=false"]
+        }
+    ]
+}
```

Comment:
Is this a shared config? Or accidental commit?

Reply:
This is a shared config, allowing Lift devs using VSCode to have a strong jest integration:
![image](https://user-images.githubusercontent.com/19605940/159026342-28b032b1-af51-4e87-b9f9-ae0742c1c21e.png)

Reply:
Got it thanks!